README.md

```diff
@@ -201,6 +201,23 @@ Tags can also be set globally using setTags
 client.setTags(['Tag1', 'Tag2']);
 ```
 
+### Timestamp
+
+You can specify the exact time your error occurred in the Send() function with the `timestamp` parameter.
+Otherwise, the current time will be used.
+
+In milliseconds since epoch:
+
+```javascript
+client.send(new Error(), { timestamp: 1718268992929 });
+```
+
+As `Date` object:
+
+```javascript
+client.send(new Error(), { timestamp: new Date(2024, 5, 13, 10, 0, 0) });
+```
```

Comment:
Is it only these 2 formats that can be used?

Reply:
Yes, I expect most users are either using `Date` or ms timestamps.